### ISSUE TITLE
Incorrect source mappings because of certain (Unicode?) characters in comments

### ISSUE THREAD

**Issue by rajeevgopalakrishna** [crytic/slither#251]
Certain characters (or scripts) in Solidity comments appear to cause incorrect source mappings.

For example, in `0x06012c8cf97bead5deae237070f9587f8e7a266d_KittyCore.sol`, the symbol that looks like underscore in "email_protected":
```
/// @author Dieter Shirley <<a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="6004051405200118090f0d1a050e4e030f">[email_protected]</a>> (https://github.com/dete) 
```
Similarly, the Asian characters in below comments from `0x5d0d76787d9d564061dd23f8209f804a3b8ad2f2_FoMo3Dlong.sol` also cause source mapping problems:

```
struct Round {
        uint256 plyr;   // pID of player in lead， lead领导吗？                                                                
        uint256 team;   // tID of team in lead                                                                                 
        uint256 end;    // time ends/ended                                                                                     
        bool ended;     // has round end function been ran  这个开关值得研究下                                                 
	uint256 strt;   // time round started                                                                                  
	uint256 keys;   // keys                                                                                                
	uint256 eth;    // total eth in                                                                                        
	uint256 pot;    // eth to pot (during round) / final amount paid to winner (after round ends)                          
        uint256 mask;   // global mask                                                                                         
        uint256 ico;    // total eth sent in during ICO phase                                                                  
        uint256 icoGen; // total eth for gen during ICO phase                                                                  
        uint256 icoAvg; // average key price for ICO phase                                                                     
    }
```

**Comment by montyly** [comment]
It seems to be fixed with #252 (merged in `master`/`dev`). @rajeevgopalakrishna can you confirm it?

**Comment by rajeevgopalakrishna** [comment]
Yes, confirm that this is fixed in contracts I have tested so far.
